### ISSUE TITLE
Add to idyll example gallery? 

### ISSUE THREAD

**Issue by mathisonian** [loki3/flexagonator#1]
Hi @loki3,

I just stumbled across your repo looking at idyll's [network graph](https://github.com/idyll-lang/idyll/network/dependents), very cool work! I wonder if you would be open to us including this on Idyll's example gallery?

Best,
Matt

p.s. - we're working on tools to better support the use case of multipage idyll articles like this one so there should be less of a need for manually renaming / copying files in the future

**Comment by loki3** [comment]
Hey Matt,
Yes, definitely.  I've been meaning to ping you or leave a message on gitter pointing people to it.
I was happy when I found Idyll because it was exactly the sort of thing I had been looking for.  Manually building HTML and JavaScript to do that sort of custom behavior just seems way too clunky.  Markdown & declarative wiring is a nice way to go.  Also glad that there will be a nicer way to create multi-page articles.
BTW, I work at Tableau Research, just down the road from you, so we have a few acquaintances in common.
Scott Sherman

    On Friday, June 7, 2019, 11:38:38 AM PDT, Matthew Conlen <notifications@github.com> wrote:  
 
 
Hi @loki3,

I just stumbled across your repo looking at idyll's network graph, very cool work! I wonder if you would be open to us including this on Idyll's example gallery?

Best,
Matt

p.s. - we're working on tools to better support the use case of multipage idyll articles like this one so there should be less of a need for manually renaming / copying files in the future
